src/olympia/addons/tests/test_views.py

```diff
@@ -1776,6 +1776,14 @@ def test_check_blocklist(self):
         assert 'Version 0.0.1 matches ' in str(response.data['non_field_errors'])
         assert self.addon.reload().versions.count() == 1
 
+    def test_cant_update_disabled_addon(self):
+        self.addon.update(status=amo.STATUS_DISABLED)
+        response = self.client.post(
+            self.url,
+            data=self.minimal_data,
+        )
+        assert response.status_code == 403
```

Comment:
I notice in other tests we also assert about `response.data`. Is that worth doing here as well?

Reply:
added extra assertions in https://github.com/mozilla/addons-server/pull/18383/commits/777df1f9cd22b2b061c4e2675b2327f4c7f95b99